docker/README.md

```diff
@@ -34,58 +33,112 @@ If you do not have or want to download the Orion repository, you can create a fi
 	  ports:
 	    - "1026:1026"
 	  command: -dbhost mongo
+```
+> Uncomment the `--smallfiles` line if you host doesn't have too much free space.
+3. Using the command-line and within the directory you created type: `sudo docker-compose up`.
```

Comment:
Check rendering... it seems that the bullet number 3 is put alongside the "grey part".